README.md

```diff
@@ -1,161 +1,76 @@
- - Run npm run build to clean directory
+Feel free to check the [issues page](https://github.com/Masuduzzaman811/Kanban-board/issues).
 
- ## Step - 4
- #### Modify webpack.config.js
- - Modify webpack.config.js to point HtmlWebpackPlugin towards your template created
+## Show your support
 
-# Loading CSS
-> In order to import a CSS file from within a JavaScript module, install and add the style-loader and css-loader to the module configuration file.
+Give a ⭐️ if you like this project!
 
-## Step - 1
-#### Setup
-- Change script on the index.html and output filename on config file from main.js to bundle.js
-- Then run and build
+## Acknowledgments
 
-## Step - 2
-- to install css loader run the following command
+- [TvMazeAPI](https://www.tvmaze.com/api)
+- [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270)
 
-```
-npm install --save-dev style-loader css-loader
-```
-- Add style loader to module config file
-
-```
-module: {
-    rules: [
-      {
-        test: /\.css$/i,
-        use: ['style-loader', 'css-loader'],
-      },
-    ],
-  },
-  ```
-
-## Using webpack-dev-server
-> The webpack-dev-server provides a rudimentary web server and the ability to use live reloading. Let's set it up:
-
-## Step - 1
-Install web dev server
-
-```
-npm install --save-dev webpack-dev-server
-```
-
-## Step - 2
-
-- Change your configuration file to tell the dev server where to look for files
-- This tells webpack-dev-server to serve the files from the dist directory on localhost:8080.
-
-## Step - 3
-> Add a script to easily run the dev server as well on the package.json file
-
-### Run 
-- Now run npm start from the command line and the browser will automatically load up the page. 
-- If any of the source files is changed and saved the web server will automatically reload after the code has been compiled. 
-
-## Acknowledgenments
-
-- [webpack's getting Started file](https://webpack.js.org/guides/getting-started/) whose instructions were used in creating the project and their notes were used to summarise and explain all the steps taken on the readme.md file.
-
-- [Setting up HtmlWebpackPlugin](https://webpack.js.org/guides/output-management/#setting-up-htmlwebpackplugin) whose instructions were used in the creation of this project and notes were used to summarise each step on the readme.md file
-
-- [Loading css](https://webpack.js.org/guides/asset-management/#loading-css) whose instructions were used in the creation of this project and notes were used to summarise each step on the readme.md file
-
-- [Using webpack-dev-server](https://webpack.js.org/guides/development/#using-webpack-dev-server) whose instructions were used in the creation of this project and notes were used to summarise each step on the readme.md file
+## 📝 License
 
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it.